src/pages/index.js

```diff
@@ -406,103 +290,81 @@ const WelcomeTile = ({
       }
 
       @media screen and (max-width: 650px) {
+        font-size: 14px;
         &:not(:last-child) {
           margin-bottom: 2rem;
         }
       }
 
-      &::before {
-        content: counter(welcome-tile);
-        counter-increment: welcome-tile;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        position: absolute;
-        top: 0;
-        left: 50%;
-        transform: translate(-50%, -50%);
-        border-radius: 50%;
-        height: 2rem;
-        width: 2rem;
-        color: var(--number-color, currentColor);
-        border: 1px solid var(--number-border-color);
-        background: var(--number-background-color);
-        z-index: 1;
-      }
-
-      &::after {
-        content: '';
-        position: absolute;
-        top: 0;
-        left: 50%;
-        transform: translate(-50%, -50%);
-        border-radius: 50%;
-        height: 2.75rem;
-        width: 2.75rem;
-        border: 1px solid
-          var(--outer-ring-border-color, var(--tile-border-color));
-        background: var(--primary-background-color);
-        transition: border-color 0.15s ease-out;
-      }
-
       &:hover {
         color: currentColor;
-
-        &::before {
-          animation: ${pulse} 1.5s infinite;
-        }
       }
-
-      ${welcomeTileStyles[variant]};
     `}
   >
-    <h3>{title}</h3>
-    <p>{description}</p>
-  </SurfaceLink>
+    <div
+      css={css`
+        display: flex;
+        padding-left: 2rem;
```

Comment:
I think we want a tiny bit of padding on both the left _and_ the right.
```suggestion
        padding: 0 2rem;
```